packages/strapi-admin/admin/src/containers/AdminPage/index.js

```diff
@@ -59,6 +59,8 @@ import selectAdminPage from './selectors';
 
 import styles from './styles.scss';
 
+const PLUGINS_TO_BLOCK_PRODUCTION = ['content-type-builder', 'settings-manager'];
```

Comment:
I think each plugins have to handle it himself instead of hard coding in the admin.

Reply:
As discussed with @Aurelsicoko and stated in the description we're going to block it from the front end for the time being.

We will block them when we develop the plugins' interdependencies